go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/DataDog/zstd v1.4.0 // indirect
 	github.com/Shopify/sarama v1.23.0
 	github.com/apache/thrift v0.0.0-20161221203622-b2a4d4ae21c7
-	github.com/aws/aws-sdk-go v1.25.34
+	github.com/aws/aws-sdk-go v1.34.13
```

Comment:
Is any of those dependencies update required for this diff? If not, probably we can revert those changes in go.mod and go.sum?

Reply:
Yes I added the v7 elasticsearch library, `github.com/olivere/elastic/v7 v7.0.21` 
Because of that adding, the go.mod and sum both changed accordingly with the command.